ietf-optical-impairment-topology.yang

```diff
@@ -1264,17 +1287,122 @@ module ietf-optical-impairment-topology {
     }
   } // augmentation connectivity-matrix
 
+  augment "/nw:networks/nw:network/nw:node/tet:te/"
+        + "tet:te-node-attributes/tet:connectivity-matrices/"
+        + "tet:connectivity-matrix/tet:from" {
+    when "../../../../../../nw:network-types/tet:te-topology/"
+       + "optical-imp-topo:optical-impairment-topology" {
+      description
+        "This augment is only valid for 
+        Optical Impairment topology ";
+    }
+    description
+      "Augment the attributes for the 'from' LTP for the TE node 
+      connectivity matrix entry.";
+    list additional-ltp {
+      when "derived-from-or-self(../../../../../../"
+         + "nt:termination-point[nt:tp-id=current()/../tet:tp-ref]/"
+         + "optical-imp-topo:protection-type,"
+         + "'optical-imp-topo:otsi-protection')" {
+        description
+          "This list applies only to 'from' LTPs that support 
+          individual OTSi(G) protection.";
+      }
```

Comment:
this list in the "from" LTPs applies only when the "to" LTP support OPS protection

Reply:
Fixed in https://github.com/ietf-ccamp-wg/draft-ietf-ccamp-optical-impairment-topology-yang/pull/116/commits/91193f0458ddb0b0bf3c4ec40e73356df546d39e